doc/index.rst

```diff
@@ -390,6 +390,14 @@ In such cases, you can specify the script(s) in `BEFORE_SCRIPT` and/or `AFTER_SC
   script:
     - .ci_config/travis.sh
 
+Multiple commands can be passed, as in a general `bash` manner. Using semi-colon as a delimitter,::
+
+    - BEFORE_SCRIPT="ls /tmp/1; ls /tmp/2"
+
+Or making a chain of commands::
+
+    - BEFORE_SCRIPT="ls /tmp/1 && ls /tmp/2 || ls /tmp/3"
+      
```

Comment:
For more complicated cases the commands should go into a dedicated script:
`BEFORE_SCRIPT='./my_before_script,sh`